dist/README.md

```diff
@@ -105,7 +136,21 @@ If the metric selected is `"type": "table"`, an example `table` response:
 ]
 ```
 
-### Annotation API
+#### Additional data
+
+Sending additional data for each metric is supported via field that allows you to enter JSON.
```

Comment:
via the Additional JSON Data input field...

Reply:
Thanks!